### PR TITLE
Support for Node v8

### PR DESCRIPTION
**Pull request by AileenCGN**
Added `package-lock.json` to `.gitignore` file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "test-travis": "istanbul cover node_modules/.bin/_mocha --report lcovonly -- $(find test -name '*.test.js')",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
+  "engines": {
+      "node": "^4.5.0 || ^6.9.0 || ^8.8.0"
```
Comment (kirrg001) [inline package.json]: 8.9 is the official LTS release version

Comment (AileenCGN) [reply package.json]: Oopsi!! Thanks, corrected it!

File: .travis.yml [hunk 2]
```diff
@@ -22,9 +23,6 @@ after_script:
   - 'npm install coveralls && cat coverage/lcov.info | coveralls'
 
 after_success:
-  - 'curl -Lo travis_after_all.py https://git.io/travis_after_all'
-  - python travis_after_all.py
-  - 'export $(cat .to_export_back) &> /dev/null'
```
Comment (kirrg001) [inline .travis.yml]: What is that?

Comment (AileenCGN) [reply .travis.yml]: This repository uses [semantic-releast](https://github.com/semantic-release/semantic-release) and this was needed for it. With version 6, it was not necessary anymore: https://github.com/semantic-release/semantic-release/releases/tag/v6.0.0

Comment (kirrg001) [reply .travis.yml]: Thanks 👍

**Pull request merged**